lib/router/README.md

```diff
@@ -0,0 +1,68 @@
+# Storybook Router
+
+Storybook Router is a wrapper library for reach/router.
+It ensures a single version of the router is used everywhere.
+
+It also includes some ready to use utils to read the path, query, viewMode and storyId from location.
+
+It also includes Components:
```

Comment:
Do you need all that documentation for properties? If you have interfaces and definitions, seems like a duplicating. Also it will be better to write JSDoc for attributes. For example, this one is not obvious from code:

```
- **children** - a function that will receive a location when it changes
```

Reply:
The same for component documentation? Why do not keep it as a part of the component?

A clickable element that will `navigate` to a new location/path.

Reply:
I guess because we link to this readme, and so a search on the readme or github will show up this useful info?

If you think this documentation is redundant, because the typings already cover this info; should I just remove it?

Reply:
I moved the docs into code comments